src/components/backend-ai-session-launcher.ts

```diff
@@ -1271,7 +1271,7 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
         config['mount_map'] = {};
         for (const f in this.folderMapping) {
           if ({}.hasOwnProperty.call(this.folderMapping, f)) {
-            config['mount_map'][f] = '/home/work/' + this.folderMapping[f];
+            config['mount_map'][f] = '/' + this.folderMapping[f];
           }
```

Comment:
For now, we've been using aliasing as renaming mounted folder in `/home/work/` subdirectory of container corresponding to session. When it comes to change this policy to mount in root(`/`) not in `/home/work/` It will bring chaos, unless we guide with an extra explanation or another option.

IMHO, we need to explicitly add the absolute path as a default input in every alias column, so that users could know that this alias would change the mount path completely.